src/components/MediaPlayer/VideoJS/components/js/VideoJSTrackScrubber.js

```diff
@@ -24,15 +25,19 @@ class VideoJSTrackScrubber extends vjsComponent {
 		this.options = options;
 		this.player = player;
 
-		/* When player is ready, call method to mount React component */
-		player.ready(() => {
-			this.mount();
-		});
+		/* When player is ready and the trackScrubber element is initialized,
+		call method to mount React component.
+		*/
+		if (this.options.trackScrubberRef.current) {
+			player.ready(() => {
+				this.mount();
+			});
```

Comment:
It looks like everything in this file is indented at least twice as far as it should be for some reason?

Reply:
It was formatting issue when I created the files. Thanks for catching that. I wasn't seeing that in my editor with its settings.